cibuild.sh

```diff
@@ -99,7 +99,7 @@ get_current_linux_name() {
         echo "centos"
         return 0
     elif [ "$(cat /etc/*-release | grep -cim1 rhel)" -eq 1 ]; then
-        if [ "$(cat /etc/os-release | grep -cim1 'VERSION_ID="7\.')" -eq 1 ]; then
+        if [ "$(grep -cim1 'VERSION_ID="7\.' /etc/os-release)" -eq 1 ]; then
```

Comment:
This is the only thing I have real questions about. It's better (fewer processes, no need for a pipe), but loses parallelism. Anyone care one way or the other?